package.json

```diff
@@ -8,6 +8,7 @@
     "url": "https://github.com/mongoosejs/studio"
   },
   "dependencies": {
+    "@mongoosejs/task": "^0.3.0",
```

Comment:
Shouldn't add this to Studio dependencies, because otherwise `@mongoosejs/task` will always be installed even if the user hasn't installed it.